test/Mono.Linker.Tests.Cases/References/AssemblyOnlyUsedByUsingWithCsc.cs

```diff
@@ -6,25 +6,20 @@ namespace Mono.Linker.Tests.Cases.References
 {
 	/// <summary>
 	/// We can't detect the using usage in the assembly.  As a result, nothing in `library` is going to be marked and that assembly will be deleted.
-	/// Because of that, `copied` needs to have it's reference to `library` removed even though we specified an assembly action of `copy`
+	/// Previously, we used to rewrite copied assemblies that had any references removed -- now copy action should leave the assembly untouched in
+	/// the output directory, even if that means having dangling references.
 	/// </summary>
 	[SetupLinkerAction ("copy", "copied")]
-	[SetupLinkerArgument ("--keep-facades", "false")]
 	[SetupCompileBefore ("library.dll", new[] { "Dependencies/AssemblyOnlyUsedByUsing_Lib.cs" })]
 
 	// When csc is used, `copied.dll` will have a reference to `library.dll`
 	[SetupCompileBefore ("copied.dll", new[] { "Dependencies/AssemblyOnlyUsedByUsing_Copied.cs" }, new[] { "library.dll" }, compilerToUse: "csc")]
 
-	// Here to assert that the test is setup correctly to copy the copied assembly.  This is an important aspect of the bug
 	[KeptMemberInAssembly ("copied.dll", typeof (AssemblyOnlyUsedByUsing_Copied), "Unused()")]
+	[KeptReferencesInAssembly ("copied.dll", new[] { "System.Private.CoreLib", "library" })]
 
-	// We library should be gone.  The `using` statement leaves no traces in the IL so nothing in `library` will be marked
 	[RemovedAssembly ("library.dll")]
-#if NETCOREAPP
-	[KeptReferencesInAssembly ("copied.dll", new[] { "System.Private.CoreLib" })]
-#else
-	[KeptReferencesInAssembly ("copied.dll", new[] { "mscorlib" })]
```

Comment:
It would be appreciated if you left the #if's for asserting mscorlib.  Unity will still be linking against an mscorlib for years to come.  Our linker is a super set of illink, which means we run and pass all of the same tests.  Dropping the #if's will introduce additional work for us to when we sync up as we'll have to add them back.

I believe someone added a constant at some point so that this code wouldn't have to use a #if?

Reply:
Use this instead: https://github.com/mono/linker/blob/main/test/Mono.Linker.Tests.Cases.Expectations/Helpers/PlatformAssemblies.cs

Reply:
> Unity will still be linking against an mscorlib for years to come. 

I wasn't aware, thanks! Keeping the mscorlib check.